test/data/split/manual_test.out

```diff
@@ -0,0 +1,10 @@
+0 start_position=890,length=150,errors=3,reference_id='1',reference_file='single_reference.fasta'	0,
+1 start_position=7601,length=150,errors=3,reference_id='1',reference_file='single_reference.fasta'	2,3,
+2 start_position=2395,length=150,errors=3,reference_id='1',reference_file='single_reference.fasta'	0,
+3 start_position=1518,length=150,errors=3,reference_id='1',reference_file='single_reference.fasta'	0,
+4 start_position=10047,length=150,errors=3,reference_id='1',reference_file='single_reference.fasta'	3,
+5 start_position=4134,length=150,errors=3,reference_id='1',reference_file='single_reference.fasta'	1,
+6 start_position=9638,length=150,errors=3,reference_id='1',reference_file='single_reference.fasta'	3,
+7 start_position=16,length=150,errors=3,reference_id='1',reference_file='single_reference.fasta'	0,
+8 start_position=9383,length=150,errors=3,reference_id='1',reference_file='single_reference.fasta'	3,
+9 start_position=1278,length=150,errors=3,reference_id='1',reference_file='single_reference.fasta'	0,
```

Comment:
This output is using the original IBF build over manually created segment files (`bin_*.fasta`)